core/frontend/src/components/bridges/BridgeCard.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+  <v-card
+    width="100%"
+    class="available-bridge pa-0 my-4"
+  >
+    <v-card-text class="pa-4">
+      <v-container class="pa-0">
+        <v-card class="elevation-0 pa-0">
+          <v-card-text
+            class="d-flex align-center pa-0 justify-center"
+          >
+            <span class="text-subtitle-1 mr-2">
+              {{ bridge.serial_path }}:{{ bridge.baud }}
+            </span>
+            <v-icon>mdi-link</v-icon>
+            <span class="text-subtitle-1 ml-2">
+              {{ bridge.ip }}:{{ bridge.udp_port }}
+            </span>
+          </v-card-text>
+        </v-card>
+      </v-container>
+      <v-fab-transition>
+        <v-btn
+          color="pink"
+          fab
+          dark
+          small
+          absolute
+          bottom
+          right
+          @click="removeBridge"
+        >
+          <v-icon>mdi-minus</v-icon>
+        </v-btn>
+      </v-fab-transition>
+    </v-card-text>
+  </v-card>
+</template>
+
+<script lang="ts">
+import axios from 'axios'
+import Vue, { PropType } from 'vue'
+import { getModule } from 'vuex-module-decorators'
+
+import BridgetStore from '@/store/bridget'
+import NotificationStore from '@/store/notifications'
+import { Bridge } from '@/types/bridges'
+import { bridget_service } from '@/types/frontend_services'
+import { LiveNotification, NotificationLevel } from '@/types/notifications'
+
+const notification_store: NotificationStore = getModule(NotificationStore)
+const bridget_store: BridgetStore = getModule(BridgetStore)
+
+export default Vue.extend({
+  name: 'BridgeCard',
+  props: {
+    bridge: {
+      type: Object as PropType<Bridge>,
+      required: true,
+    },
+  },
+  methods: {
+    async removeBridge(): Promise<void> {
+      bridget_store.setUpdatingBridges(true)
+      await axios({
```

Comment:
I believe that you don't need to await if you don't need to wait it to finish after this call.

Reply:
> Baudrate should be a combobox The same thing should be applied to the serial port showing a list of valid ports We should also get a default port for the user to use like: 15000 The user should also be able to change between udp/tcp server and client. ![image](https://user-images.githubusercontent.com/1215497/135357111-a72c8775-6cd0-472f-b7f5-78369fb53b97.png)

1. About baudrate, agree. Will change.
2. About serial port, I can do it, but it will show all available, inclusing autopilot, ok?
3. About default port, ok.
4. About UDP/TCP, isn't bridges UDP only? It says so on both the documentation and help menu.

Reply:
> 4\. About UDP/TCP, isn't bridges UDP only? It says so on both the documentation and help menu.

Yep, it's ok to be only udp for now.

Reply:
The original questions was also not answered.